src/promptflow/tests/sdk_cli_test/conftest.py

```diff
@@ -8,9 +8,11 @@
 import pytest
 from mock import mock
 from pytest_mock import MockerFixture
+from sdk_cli_azure_test.recording_utilities import is_live, is_record, is_replay
```

Comment:
can we make this import better? kind of wired to import a azure fixture in a local test conft...

Reply:
Okay